DYNAMIC_ARCH is broken with aarch64

With OpenBLAS v0.3.10, cross-compiling for `aarch64` is broken when `DYNAMIC_ARCH` is enabled:

```
export CC=aarch64-linux-gnu-gcc
export CXX=aarch64-linux-gnu-g++
cmake .. \
    -DCMAKE_SYSTEM_NAME=Linux \
    -DCMAKE_CROSSCOMPILING=TRUE \
    -DCMAKE_SYSTEM_PROCESSOR=aarch64 \
    -DNOFORTRAN=TRUE \
    -DDYNAMIC_ARCH=TRUE
...

make
...

[ 57%] Building ASM object kernel/CMakeFiles/kernel_CORTEXA53.dir/CMakeFiles/sgemm_otcopy_CORTEXA53.S.o
/home/paleozogt/Development/OpenBLAS/.build/kernel/CMakeFiles/sgemm_otcopy_CORTEXA53.S:8:10: fatal error: /home/paleozogt/Development/OpenBLAS/kernel/arm64/sgemm_tcopy_4.S: No such file or directory
 #include "/home/paleozogt/Development/OpenBLAS/kernel/arm64/sgemm_tcopy_4.S"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
kernel/CMakeFiles/kernel_CORTEXA53.dir/build.make:1107: recipe for target 'kernel/CMakeFiles/kernel_CORTEXA53.dir/CMakeFiles/sgemm_otcopy_CORTEXA53.S.o' failed
```

With `DYNAMIC_ARCH` disabled it builds fine.

--- a/cmake/prebuild.cmake
+++ b/cmake/prebuild.cmake
@@ -195,8 +195,13 @@ if (DEFINED CORE AND CMAKE_CROSSCOMPILING AND NOT (${HOST_OS} STREQUAL "WINDOWSS
       "#define HAVE_VFP\n"
       "#define HAVE_NEON\n"
       "#define ARMV8\n")
+if ("${TCORE}" STREQUAL "CORTEXA57")     
     set(SGEMM_UNROLL_M 16)
     set(SGEMM_UNROLL_N 4)
+else       
+    set(SGEMM_UNROLL_M 8)
+    set(SGEMM_UNROLL_N 8) 
+endif
     set(DGEMM_UNROLL_M 8)
     set(DGEMM_UNROLL_N 4)
     set(CGEMM_UNROLL_M 8)

--- a/cmake/prebuild.cmake
+++ b/cmake/prebuild.cmake
@@ -198,10 +198,10 @@ if (DEFINED CORE AND CMAKE_CROSSCOMPILING AND NOT (${HOST_OS} STREQUAL "WINDOWSS
 if ("${TCORE}" STREQUAL "CORTEXA57")     
     set(SGEMM_UNROLL_M 16)
     set(SGEMM_UNROLL_N 4)
-else       
+else ()
     set(SGEMM_UNROLL_M 8)
     set(SGEMM_UNROLL_N 8) 
-endif
+endif ()
     set(DGEMM_UNROLL_M 8)
     set(DGEMM_UNROLL_N 4)
     set(CGEMM_UNROLL_M 8)

